project/plugin.sbt

```diff
@@ -15,7 +15,7 @@ def printAndDie(msg: String): Nothing = {
 // Update SCALAJS_VERSION in release.sh, as well
 val scalaJSVersion = env("SCALAJS_VERSION") match {
   case Some("0.6.32") | None => "0.6.32"
-  case Some("1.0.1") => "1.0.1"
+  case Some("1.1.0") => "1.1.0"
```

Comment:
This is not consistent with the other changes in the build. `None` should give `1.1.0`, not `0.6.32`.

Fixing this fixes the failure with `WasPublicBeforeTyper`.

Reply:
Thanks for catching this.